src/windowsdesktop/src/windowsdesktop/src/bundle/Wix.targets

```diff
@@ -345,18 +264,25 @@
       <WixpackOutputDir Condition="'$(WixCommandPackagesDir)'!=''">$(WixCommandPackagesDir)</WixpackOutputDir>
       <WixpackOutputDir Condition="'$(WixCommandPackagesDir)'==''">$(ArtifactsPackagesDir)NonShipping</WixpackOutputDir>
       <DefaultBundleWixpackFile>$(WixpackOutputDir)\$(TargetFileName).wixpack.zip</DefaultBundleWixpackFile>
-      <BundleShippingName Condition="'$(BundleShippingName)'==''">$(OutputName).exe</BundleShippingName>
-      <!-- Expected wixpack file should match the bundle executable name -->
-      <ExpectedBundleWixpackFile>$(WixpackOutputDir)\$(BundleShippingName).wixpack.zip</ExpectedBundleWixpackFile>
-      <!-- PDB name follows Wix default: TargetName + .wixpdb -->
-      <_BundlePdbFile>$(TargetDir)$(TargetName).wixpdb</_BundlePdbFile>
+  <BundleShippingName Condition="'$(BundleShippingName)'==''">$(OutputName).exe</BundleShippingName>
+  <!-- Use version properties directly from VersionDetails.props -->
+  <!-- Use direct version properties from Version.Details.props -->
+  <WinFormsAndWpfVersion>$(MicrosoftNETCoreAppRefVersion)</WinFormsAndWpfVersion>
+  <MajorVersion>$([System.Text.RegularExpressions.Regex]::Match('$(MicrosoftNETCoreAppRefVersion)', '^(\d+)').Groups[1].Value)</MajorVersion>
```

Comment:
Assuming Major.Minor.Patch is the windowsdesktop Major.Minor.Patch, this should be eliminated. It's part of the imported `eng/Versions.props`.